src/api/projects.js

```diff
@@ -0,0 +1,62 @@
+'use strict';
+import {User} from '../db/models/user';
+import {Project} from '../db/models/project';
+import Permission from '../util/permission';
+
+/**
+ * Create project under user's organization
+ * @param  {object} event
+ * @param  {object} context
+ * @param  {object} callback
+ * @return {function} done
+ */
+export const create = (event, context, callback) => {
+  const done = (error, res) => callback(null, {
```

Comment:
@singhabahu  Can't we get this method out and generalize to other apis as well ?

Reply:
@Rishanthakumar This will be handled in this change.https://github.com/singhabahu/serverless-backend/projects/1#card-10227338

Reply:
@Rishanthakumar please refer to the other PR and confirm it